File: minwook/7주차/과일_장수.java
import java.util.*;
class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        Arrays.sort(score); // 오름차순 (내림차순이 더 편한데 int[] 구현 복잡)
        for(int i=score.length-m; i>=0; i-=m){
            answer += score[i] * m;
        }
        return answer;
    }
}

File: jeonghyeon/7week/과일 장수.js
function solution(k, m, score) {
    score.sort((a,b) => b - a); // 내림차순으로 정렬
    let boxnum = parseInt(score.length / m); // 만들 수 있는 박스 개수
    let maxprofit = 0; // 최대 이익
    for(let i = 0; i < boxnum; i++) {
        let includenum = score.slice(i * m, (i + 1) * m); // 박스에 포함 될 점수
        let minnum = Math.min(...includenum); // 현재 박스 점수들 중 가장 작은 수
        let currentbox = minnum * m; // 현재 박스 이익
        maxprofit += currentbox; // 모든 이익들 더하기
    }
    return maxprofit;
}